lib/media.js

```diff
@@ -114,48 +114,69 @@ function wrapWithTags(string, tag) {
  * @param  {string|null} tag   The type of tag to wrap the contents in.
  * @return {string}
  */
-function combineFileContents(fileArray, directory, filePrefix, tag) {
+function combineFileContents(fileArray, directory, filePrefix) {
   if (!fileArray || !fileArray.length) {
     return false;
   }
 
   // If there are files, retrieve contents
-  return getContentsOfFiles(fileArray, directory, filePrefix)
-    .then(function (fileContents) {
-      return wrapWithTags(fileContents.join(''), tag);
-    });
+  return getContentsOfFiles(fileArray, directory, filePrefix);
 }
 
 /**
  * Append at the bottom of the head tag, or if no head tag, then the top of root tag.
  *
  * @param {array} styles
+ * @param {string} fonts
  * @param {string} html
+ * @param {string} tag
  * @returns {string}
  */
-function appendMediaToTop(styles, html) {
-  var index = findTop(html);
+function appendMediaToTop(styles, fonts, html, tag) {
+  var index = findTop(html),
+    combinedStyles = fonts + styles.join(''),
+    wrappedStyles = wrapWithTags(combinedStyles, tag);
 
-  return splice(html, index, styles);
+  return splice(html, index, wrappedStyles);
 }
 
 /**
  * Append at the bottom of the body tag, or if no body tag, then the bottom of the root tag.
  * @param {Array} scripts
  * @param {string} html
+ * @param {string} tag
  * @returns {string}
  */
-function appendMediaToBottom(scripts, html) {
-  var index = findBottom(html);
+function appendMediaToBottom(scripts, html, tag) {
+  var index = findBottom(html),
+    wrappedScripts = wrapWithTags(scripts.join(''), tag);
+
+  return splice(html, index, wrappedScripts);
+}
 
-  return splice(html, index, scripts);
+/**
+ * Get the fonts for the site to be inlined
+ * @param  {Object} site
+ * @return {Promise}
+ */
+function getFonts(site) {
+  var pathToFontCSS = path.join(currentDir, site.assetDir, 'fonts', `combined-${site.slug}.css`);
+
+  // Return a promise for reading the file's contents
+  return files.readFilePromise(pathToFontCSS)
+    .catch(function () {
+      // If there was an error reading from the file
+      // because it doesn't exist, return an empty string
+      return '';
```

Comment:
Would it be useful if it output a CSS comment perhaps? Maybe for debugging purposes?